packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -97,6 +97,36 @@ When using red text for destructive actions, it is still recommended to require
 
 <img src="./img/red-delete-text.png" alt="drilldown menu" width="935"/>
 
+## Disabling menus and menu options
+Menus or menu items can be disabled for multiple reasons. The reason an action is disabled informs which design pattern you should use.
+
+### **An action is unavailable due to an unmet prerequisite, condition, permission, or status.** 
+If a user needs to complete a prerequisite to enable an action, disable the action and add a [tooltip](components/tooltip). The tooltip should explain what the user needs to do to enable the action. 
+
+<img src="./img/dropdown-disabled-action-tooltip.png" alt="disabled action tooltip" width="935"/>
+
+Example: A user can’t perform bulk actions until they select resources in the list.
+
+Example: A user with view-only privileges looks at a Pod’s details page and the options within the Actions dropdown are disabled.
+
+If an action is dependent on an unmet condition or status in the console, disable the action with a [tooltip](components/tooltip) explaining why the action currently is unavailable.
+
+Example: A user cannot view past logs until their container is finished restarting.
+
+### **An action cannot be performed due to a product constraint or rule.**
+If an action cannot be taken because of a product constraint or rule, hide the action. 
+
+Example: While a user can delete a resource they own, they cannot delete a template or default resource.
```

Comment:
This example makes sense, but the one below about permissions doesn't make sense in this section since you're already mentioning the permission use case in the first mini header? So I would just leave the first example and delete the second